src/hooks/scrollRemember.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+
+interface UseScrollRememberReturns {
+  resumePosition(): void,
+}
+
+export const useScrollRemember = (): UseScrollRememberReturns => {
+  const router = useRouter()
+
+  // Store last scroll position so we can return to it when we load the page next time
+  useEffect(() => {
+    const handler = () => {
+      localStorage.setItem('lastScroll/' + router.pathname, window.pageYOffset.toString())
```

Comment:
Does this list position persist even if I navigate away from the app and come back to it, and is this the behaviour we want? What would be the effects of using session storage or sharing state between page navigations?

Reply:
Yes, this would happen. I can try to hook onto some 'leavePage' event and clear it, maybe?